subprojects/docs/src/docs/release/notes.md

```diff
@@ -101,6 +101,22 @@ This means that you can query for the Javadoc or sources _variant_ of a module a
 This also works in multi-projects.
 If activated, a Java and Java Library project automatically provides the `javadocJar` and `sourcesJar` tasks.
 
+## Problems with task definitions called out during build
```

Comment:
Should this section be moved to [features for plugin authors](https://docs.gradle.org/release-nightly/release-notes.html#features-for-plugin-authors)

Reply:
It's note exclusively for plugin authors, so I'd leave it here.